### ISSUE TITLE
It's possible to create a message with the field 'timestamp'

### ISSUE THREAD

**Issue by alanjcastonguay** [alanjcastonguay/logstash-output-loginsight#6]
Log Insight will refuse events with a "timestamp" field. Remap it.

**Comment by alanjcastonguay** [comment]
This could be avoided by specifying `adjusted_fields` in the configuration to rewrite the `timestamp` field. I'm going to resolve this ticket by changing the default `adjusted_fields` as follows.

```
:adjusted_fields =>
{
    "hostname" => "host",
    "host" => "hostname",
    "@version" => nil,
    "@timestamp" => nil,
    "message" => nil,
    "timestamp" => "timestamp_",
  }
```

**Comment by alanjcastonguay** [comment]
Resolved in 8426dff53c0271816f4bbc0896c07bc08c809b7c
